src/main/java/software/amazon/cloudwatchlogs/emf/serializers/StorageResolutionFilter.java

```diff
@@ -0,0 +1,15 @@
+package software.amazon.cloudwatchlogs.emf.serializers;
+
+import software.amazon.cloudwatchlogs.emf.model.StorageResolution;
+
+public class StorageResolutionFilter {
+
+    @Override
+    public boolean equals(Object obj) {
+
+        if (obj == null || !(obj instanceof StorageResolution)) {
```

Comment:
Unnecessary null check

Reply:
Why do we need this filter class? Even if user is not using the storage resolution set, but we can still pass it with StorageResolution:60 in the parsed emf json. Looking at the constructors of MetricDefinition class, we always forcing StorageResolution.Standard. I believe we also need Deserializer for StorageResolution.

Please correct me if I am missing anything here

Reply:
> Is this filter class required? Assuming if user is not using the storage resolution set, but we can still pass it with StorageResolution:60 in the parsed emf json. Looking at the constructors of MetricDefinition class, we always forcing StorageResolution.Standard. This comment is more of a discussion.

Yes it is required because we have decided that we will not serialize storage resolution if the value is 60. It will unnecessarily increase the payload size.


> Are we missing Deserialization annnotation for StorageResolution?

As per my observation, we are not deserializing anywhere. Hence, I didn't add that annotation. I suppose you mentioned this because 'Unit' parameter also has Deserialization annotation. However, I didn't find any use of it so I ignored it for the new parameter (StorageResolution).

Let me know what's your suggestion on how should we proceed.

Reply:
> Unnecessary null check

Thanks for pointing this out. Updated!